server/apps/distributed-app/docs/modules/ROOT/pages/configure/redis.adoc

```diff
@@ -18,6 +18,9 @@ to get some examples and hints.
 | redis.topology
 | Redis server topology. Defaults to standalone. Possible values: standalone, cluster, master-replica
 
+| redis.readFrom
+| The property to determine how Lettuce routes read operations to Redis server with topologies other than standalone. Defaults to master. Possible values: master, masterPreferred, replica, replicaPreferred, any
```

Comment:
add a reference to lettuce values IMO

Reply:
BTW can you add this entry in an example `redis.properties` too?

Reply:
done